pkg/apis/clusteroperator/types.go

```diff
@@ -227,10 +227,10 @@ type OpenShiftConfigVersion struct {
 
 // ClusterDeploymentSpec is the specification of a cluster's hardware and configuration
 type ClusterDeploymentSpec struct {
-	// ClusterID is the ID to assign to the cluster in the cloud provider.
+	// ClusterName is the name to assign to the cluster in the cloud provider.
 	// This will be generated by the API Server when the ClusterDeployment is
 	// created. It should not be set by the user.
-	ClusterID string
+	ClusterName string `json:"clusterName"`
```

Comment:
In the unversioned API, do not add the json tag to field. In other words, remove the `` `json:"clusterName"`` `

Reply:
@staebler ah, ok, thanks for clarifying that. I fixed it for NetworkingConfig as well in the same file.